recipes/tinydir/all/conanfile.py

```diff
@@ -0,0 +1,25 @@
+import os
+from conans import ConanFile, tools
+
+
+class TinydirConan(ConanFile):
+    name = "tinydir"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/cxong/tinydir"
+    description = "Lightweight, portable and easy to integrate C directory and file reader"
+    topics = ("conan", "c", "portable", "filesystem", "directory", "posix", "header-only")
```

Comment:
I believe this is missing

```suggestion
    topics = ("conan", "c", "portable", "filesystem", "directory", "posix", "header-only")
    no_copy_source = True
```

Reply:
I think it could be added, I'll update my templates